packages/core/src/browser/tree/test/mock-tree-model.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2018 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { TreeNode, CompositeTreeNode } from '../tree';
+import { SelectableTreeNode } from '../tree-selection';
+import { ExpandableTreeNode } from '../tree-expansion';
+
+export interface Node {
```

Comment:
It will add one more `Node` type to workspace symbols, please use something more specific to make it findable